src/types.ts

```diff
@@ -12,6 +21,33 @@ export interface BidPostDto {
   expireBlock: string;
 }
 
+export interface BidParams {
+  account: string;
+  auctionId: string;
+  bidAmount: string;
+  contractAddress: string;
+  tokenId: string;
+  minimumBid: string;
+  startBlock: string;
+  expireBlock: string;
+}
+
+export interface BidsList {
+  [nftId: string]: Bid[] | undefined;
+}
+
+export interface BidsListDto {
+  nftIds: string[];
+}
+
+export interface BidsAccountsDto {
+  account: string;
+}
+
+export interface BidsDto {
+  nftId: string;
+}
+
 export interface Bid {
```

Comment:
Nit: You could make this
`export interface Bid extends BidParams`
to make use of the `BidParams` interface you made earlier